core/Module.mustache

```diff
@@ -1,20 +1,251 @@
+
+### Example: Navigation Bar
+
+The `OudsNavigationBar` manages the bottom navigation. It works in tandem with `OudsNavigationBarItem`.
+
+```kotlin
+import com.orange.ouds.core.component.OudsNavigationBar
+import com.orange.ouds.core.component.OudsNavigationBarItem
+
+@Composable
+internal fun OudsNavigationBarSample() {
+    data class Item(val label: String, val imageVector: ImageVector)
+    val items = listOf(
+        Item("Call", Icons.Default.Call),
+        Item("Email", Icons.Default.Email),
+        Item("Settings", Icons.Default.Settings)
+    )
+    var selectedItemIndex: Int by rememberSaveable { mutableIntStateOf(0) }
+
+    OudsNavigationBar(
+        items = items.mapIndexed { index, item ->
+            OudsNavigationBarItem(
+                selected =  index == selectedItemIndex,
+                onClick = { selectedItemIndex = index },
+                icon = OudsNavigationBarItemIcon(imageVector = item.imageVector),
+                label = item.label,
+                // Optional: Add a badge
+                badge = if (index == 1) OudsNavigationBarItemBadge(contentDescription = "Unread emails", count = "2") else null
+            )
+        }
+    )
+}
+```
+
+### Available components and versions
 
 <table>
-{{#versions}}
-<tr><td>{{name}}</td><td>{{value}}</td></tr>
-{{/versions}}
+    {{#versions}}
+        <tr>
+            <td>{{name}}</td>
+            <td>{{value}}</td>
+        </tr>
+    {{/versions}}
 </table>
 
+## Best Practices
+
+1. **Do not mix Material 3 components directly with OUDS components** if an OUDS equivalent exists. Using the native components might result in incorrect colors
+or spacing that violates the brand guidelines.
+2. **Use `OudsTheme` tokens** for custom layouts. If you need to build a custom card or row, use OudsTheme.colorScheme instead of Color.Black or Color.White to
+ensure dark mode compatibility.
+3. **Window Insets:** The library is designed to work edge-to-edge. Components like OudsNavigationBar can be transparent to allow content to blur behind them (
+if supported), but they require proper handling of WindowInsets in your layouts.
+
 # Package com.orange.ouds.core.component
 
-Contains all OUDS basic components: OudsButton, OudsLink,...
+This package contains the catalog of ready-to-use UI components (like OudsButton, OudsNavigationBar) that implement the design system specifications using
+Jetpack Compose.
 
 # Package com.orange.ouds.core.theme
 
-Contains the `OudsTheme` composable that can be used in your app to apply a chosen theme. The other files are used to build the theme and don't have to be used
-directly but only through the `OudsTheme`.
\ No newline at end of file
+This package contains the logic for the OudsTheme composable, including the OudsThemeTweak mechanism and the implementation of the theme contract that injects
+visual data into the application.
+
+# Package com.orange.ouds.core.utilities
```

Comment:
Packages `com.orange.ouds.core.component.common` and `com.orange.ouds.core.extensions` also appear in the generated documentation but there is no description. But maybe their content should be internal or moved to another package.